validator/src/main.rs

```diff
@@ -2006,6 +2006,14 @@ pub fn main() {
                 .takes_value(true)
                 .help("Number of bins to divide the accounts index into"),
         )
+        .arg(
+            Arg::with_name("accounts_index_path")
+                .long("accounts-index-path")
+                .value_name("PATH")
+                .takes_value(true)
+                .multiple(true)
+                .help("Persistent accounts-index location"),
```

Comment:
Would it be valuable to include in the help text that this option may be specified multiple times? A few of the options that accept multiple say "May be specified multiple times.". And maybe what the default value is too?

Reply:
good point. I added them.